docs/guide/en/result.md

```diff
@@ -128,19 +128,29 @@ Returning to the previous example, when `name` and `email` belong to a `user` at
 ];
 ```
 
-Also keep in mind that attribute names are always strings, even when used with `Each`:
+Also keep in mind that attribute names must be strings, even when used with `Each`:
 
 ```php
 $rule = new Each([new Number(min: 21)]),
 ```
 
-Given `[21, 22, 23, 20]` input, the output will be: 
+Given `[21, 22, 23, 20]` input throw exception, because default array keys is int:
+
+```php
+InvalidArgumentException
+Top level attributes can only have string type.
+```
```

Comment:
Given `[21, 22, 23, 20]` input throw `InvalidArgumentException` with message "Top level attributes can only have string type.", because default array keys is integer.